faq.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta charset="utf-8" />
-  <title>AntennaPod | Knowledge Base</title>
+  <title>AntennaPod | F.A.Q.</title>
```

Comment:
So I'll leave FAQ for now, mainly for consistency with the app. Can change back once it's extended.